morpheus/_lib/src/stages/preprocess_nlp.cpp

```diff
@@ -55,9 +52,9 @@ PreprocessNLPStage::PreprocessNLPStage(const neo::Segment &parent,
   m_stride(stride)
 {}
 
-PreprocessNLPStage::operator_fn_t PreprocessNLPStage::build_operator()
+PreprocessNLPStage::subscribe_fn_t PreprocessNLPStage::build_operator()
 {
-    return [this](neo::Observable<reader_type_t> &input, neo::Subscriber<writer_type_t> &output) {
+    return [this](rxcpp::observable<sink_type_t> input, rxcpp::subscriber<source_type_t> output) {
```

Comment:
Great to see these changes; makes everything simpler.

Reply:
Ya much better.